src/components/UpupUploader/FileBrowser/ListItem.tsx

```diff
@@ -51,33 +51,53 @@ const ListItem = ({
         ...(isFileSelected && { backgroundColor: backgroundColors.selected }),
         ...(!isFileSelected && { backgroundColor: backgroundColors.default }),
     }
-    const initial: Target = {
+
+    const transition: AnimationProps['transition'] = {
+        type: 'spring',
+        damping: 10,
+        stiffness: 100,
+        opacity: { delay: index * 0.05 },
+        y: { delay: index * 0.05 },
+    }
```

Comment:
Is it better to declare this in component scope rather than file gloabal file scope

Reply:
will refactor the code another time.